pkg/build/strategies/dockerfile/dockerfile.go

```diff
@@ -365,14 +368,48 @@ func getDestination(config *api.Config) string {
 	return destination
 }
 
-// checkValidDirWithContents returns true if the parameter provided is a valid,
-// accessible and non-empty directory.
-func checkValidDirWithContents(name string) (bool, []os.FileInfo) {
+// getImageScriptsDir returns the directory containing the builder image scripts
+func getImageScriptsDir(config *api.Config) (string, bool) {
+	if strings.HasPrefix(config.ScriptsURL, "image://") {
+		return strings.TrimPrefix(config.ScriptsURL, "image://"), true
```

Comment:
If the `ScriptsURL` points to an image location, indicate that we blind-trust all scripts are provided